src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -236,13 +235,13 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
     openSideSheetState.workflowStatusBarOpen,
     fullWindowWidth
   );
-  let contentBottomOffsetInPx;
+  const contentBottomOffsetInPx = 0;
```

Comment:
Let's remove this contentBottomOffsetInPx variable entirely since it's not being used anymore.